### ISSUE TITLE
End of round never triggers again after completing a full match

### ISSUE THREAD

**Issue by BenGoulet** [symbol24/ProjectJuice#159]
Whenever a full match ends and the players either Rematch or go back to Main Menu, they will notice that the next round they play will never end, since the Round Results screen will never appear again.

**Comment by symbol24** [comment]
CNR, find me some better repros, i can finish a match.

**Comment by BenGoulet** [comment]
Finishing the match isn't the issue, it's the next match's first round that will never end.

**Comment by symbol24** [comment]
Then let me rephrase, after the first match, the next first round ends.

**Comment by BenGoulet** [comment]
That's frustrating; we really thought we nailed that one's repro :/
We'll go back on it next time.

**Comment by BenGoulet** [comment]
It was found to only occur in Jeff's scene.

**Comment by BenGoulet** [comment]
This was fixed when selecting rematch, but still occurs if the players decide to go back to the main menu at the end of a match and start a new match.

**Comment by jeffcerto** [comment]
checked and fixed
